src/foundation.cs

```diff
@@ -3702,6 +3702,15 @@ public interface NSLocale : NSSecureCoding, NSCopying {
 		
 		[Internal, Field ("NSLocaleAlternateQuotationEndDelimiterKey")]
 		NSString _AlternateQuotationEndDelimiterKey { get; }
+
+		[Watch (3, 0), TV (10, 0), Mac (10, 12), iOS (10, 0)]
```

Comment:
The rest of these kind of NSLocale bindings are in src/Foundation/NSLocale.cs. Was not sure if I should do it here since it's a "new" API instead of the indirect key lookup or there.